app/dashboard/static/js/build.js

```diff
@@ -156,9 +155,9 @@
         {name: 'kci-boots-all'},
         {name: 'kci-boots-all-job'},
         {name: 'kci-boots-all-job-kernel-defconfig'},
+        {name: 'kci-boots-all-job'},
```

Comment:
This line should not be added here, the commit is to remote the boots-board-job view.

Reply:
those lines are not in this PR.

Reply:
It's in the first patch, and I guess it's undone in the second one.  Please fix this.